applications/asset_tracker_v2/src/modules/Kconfig.data_module

```diff
@@ -169,6 +169,14 @@ config DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT
 	  data in sample requests sent to other modules. This configuration can be overwritten by
 	  changing the application's real-time configuration using the cloud-side state.
 
+config DATA_SAMPLE_WIFI_DEFAULT
+	bool "Include Wi-Fi APs in sample requests"
+	default y
```

Comment:
This is problematic to be `y` by default and should likely be `n` because WiFi is not always configured into the build. We should likely set this on only when WiFi is enabled in the build. Otherwise we request `APP_DATA_LOCATION` with non-WiFi configuration even if GNSS and neighbor_cell are disabled.

Reply:
This will be handled elsewhere in `location_module.c:start_search()`